app/templates/components/event-card.hbs

```diff
@@ -17,18 +17,37 @@
           @src={{if this.event.thumbnailImageUrl this.event.thumbnailImageUrl this.event.originalImageUrl}} />
       </a>
     {{/unless}}
-    <a class="main content" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.identifier}}">
+    <a style="min-height:165px" class="main content" href="{{href-to (if this.isCFS 'public.cfs' 'public') this.event.identifier}}">
       <SmartOverflow @class="header">
         {{this.event.name}}
       </SmartOverflow>
       <div class="meta">
         <span class="date">
-          {{general-date this.event.startsAt tz=this.event.timezone}}
+          {{#if (eq (general-date this.event.startsAt 'date-short') (general-date this.event.endsAt 'date-short')) }}
```

Comment:
No, don't do it like this

Reply:
Instead check for whether it ```isSingleDay``` in event-card.js file and then use it here

Reply:
okay, just wanted to know is this wrong?